documentation/helpers/ArgTypes/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Tabs } from '@spark-ui/tabs'
+import { ArgTypes as StorybookArgTypes } from '@storybook/blocks'
+import type { FC } from 'react'
+
+interface Props<T> {
+  of?: T
+  subcomponents?: Record<string, T>
+}
+
+export const ArgTypes = <T extends FC>({ of, subcomponents = {} }: Props<T>) => {
+  const componentsList = {
```

Comment:
I kindly disagree here... For me a list is just like a "collection" and kind of _unformatted_... Renaming it would bring unnecessary complexity on a simple naming, to me.

Reply:
> Elseway I would have declared specificly something like componentsArray

That is exactly my point, `componentsList` makes you think you are dealing with an array, when it's not the case 😅

Reply:
I think we're misunderstanding each other :sweat_smile: Maybe we could discuss this specific point directly in Slack, wdyt?

Reply:
I also would think it is an array and not a record/dictionnary. When I see `componentsList` I think I can do `componentsList.map()` but I can't it would throw an error.

Personally I don't like to specify "array" "record" "list" at the end of my variables, so I would have called it simply `components`. But I also prefer what @acd02 suggested than "componentsList" for an object.

Reply:
Allright then, changed for `components` instead :+1: